Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationCode.bind

```diff
@@ -0,0 +1,85 @@
+// ## XAML - Text + buttons (DataTemplate)
+
+<DataTemplate x:Key="InAppNotificationWithButtonsTemplate">
+    <UserControl>
+        <Grid DataContext="{Binding ElementName=CurrentPage}">
+            <VisualStateManager.VisualStateGroups>
+                <VisualStateGroup>
+                    <VisualState x:Name="NarrowState">
+                        <VisualState.Setters>
+                            <Setter Target="TextBlock.(Grid.RowSpan)" Value="1" />
+                            <Setter Target="TextBlock.(Grid.ColumnSpan)" Value="2" />
+
+                            <Setter Target="ButtonsStackPanel.(Grid.Row)" Value="1" />
+                            <Setter Target="ButtonsStackPanel.(Grid.RowSpan)" Value="1" />
+                            <Setter Target="ButtonsStackPanel.(Grid.Column)" Value="0" />
+                            <Setter Target="ButtonsStackPanel.(Grid.ColumnSpan)" Value="2" />
+                            <Setter Target="ButtonsStackPanel.Orientation" Value="Vertical" />
+                            <Setter Target="ButtonsStackPanel.Margin" Value="0 5 0 0" />
+
+                            <Setter Target="YesButton.Width" Value="Auto" />
+                            <Setter Target="YesButton.HorizontalAlignment" Value="Stretch" />
+
+                            <Setter Target="NoButton.Width" Value="Auto" />
+                            <Setter Target="NoButton.HorizontalAlignment" Value="Stretch" />
+                            <Setter Target="NoButton.Margin" Value="0 5 0 0" />
+                        </VisualState.Setters>
+                                    
+                        <VisualState.StateTriggers>
+                            <StateTrigger IsActive="{Binding IsRootGridActualWidthLargerThan700, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}" />
+                        </VisualState.StateTriggers>
+                    </VisualState>
+
+                    <VisualState x:Name="LargeState">
+                        <VisualState.StateTriggers>
+                            <StateTrigger IsActive="{Binding IsRootGridActualWidthLargerThan700, Mode=OneWay}" />
+                        </VisualState.StateTriggers>
+                    </VisualState>
+                </VisualStateGroup>
+            </VisualStateManager.VisualStateGroups>
+
+            <Grid x:Name="RootGrid" SizeChanged="RootGrid_SizeChanged">
```

Comment:
My thoughts on this are to have a simple template that doesn't complicate the mechanics of the control and shows the minimum needed to try the control. We could always have both?